thirdparty_builtin/CMakeLists.txt

```diff
@@ -57,18 +57,36 @@ if(ENABLE_TESTS)
         list(APPEND gtest_defines "-DGTEST_HAS_DEATH_TEST=0")
     endif()
 
+    #
+    # Use BLT's ENABLE_GMOCK option to set the build options for 
+    # gtest's cmake project
+    #
+    
+    if(ENABLE_GMOCK)
```

Comment:
It's the way that googletest's cmake project works.

First, I don't belive you can build gmock w/o gtest. 

You either build gmock (using their `BUILD_GMOCK=ON` which also builds gtest), or you build gtest (and don't build gmock)

I haven't checked, so I don't know if you can actually set both to OFF (for a future case where we would also want to support our own`ENABLE_GTEST` -- but we could always simply not include the projects as our own guard.